sources/packages/backend/libs/test-utils/src/factories/location-fake.ts

```diff
@@ -0,0 +1,28 @@
+import * as faker from "faker";
+import { Institution, InstitutionLocation } from "@sims/sims-db";
+import { createFakeInstitution } from "@sims/test-utils";
+
+export function createFakeLocation(
```

Comment:
mm... we do have a `createFakeInstitutionLocation`, `sources/packages/backend/libs/test-utils/src/factories/institution-location.ts` right? so, do we need this?

Reply:
Leftover, removing it.

Reply:
Removed